recipes/google-cloud-container-feedstock/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "google-cloud-container" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 566834ef43e79917b112e3bd2848e84cfb0f4d7b565581607e2548f5c8e5d87a
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --verbose"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - google-api-core>=1.6.0,<2.0.0dev
+    - grpcio>=1.8.2
+    - googleapis-common-protos>=1.5.3,!=1.5.4,<2.0dev
+    - protobuf>=3.4.0
+    - google-auth>=0.4.0,<2.0dev
+    - requests>=2.18.0,<3.0.0dev
+    - setuptools>=34.0.0
+    - six>=1.10.0
+    - pytz
+    - futures>=3.0.5
+
+test:
+  imports:
+    - google.cloud
+    - google.cloud.container_v1
+    - google.cloud.container_v1.proto
+  requires:
+    - coverage
```

Comment:
Ate these needed for the import tests?

Reply:
python packages removed from test.requires